pkgs/by-name/ya/yafc-ce/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  SDL2,
+  SDL2_image,
+  SDL2_ttf,
+}:
+buildDotnetModule rec {
+  pname = "yafc-ce";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "shpaass";
+    repo = "yafc-ce";
+    rev = version;
+    hash = "sha256-TihRIRD87a1bihIM+xj+bC6MSRT/8k8oiAGquuZpWBs=";
+  };
+
+  projectFile = "Yafc/Yafc.csproj";
+  nugetDeps = ./deps.nix;
+  dotnet-sdk = dotnetCorePackages.dotnet_8.sdk;
+  dotnet-runtime = dotnetCorePackages.dotnet_8.runtime;
+  executables = "Yafc";
```

```suggestion
  executables = [ "Yafc" ];
```

Other .NET packages seem to use a list of string for this attr.